apps/design/frontend/src/api.ts

```diff
@@ -67,6 +67,38 @@ export const getElection = {
   },
 } as const;
 
+export const getElectionInfo = {
+  queryKey(id: ElectionId): QueryKey {
+    return ['getElectionInfo', id];
+  },
+  useQuery(id: ElectionId) {
+    const apiClient = useApiClient();
+    return useQuery(this.queryKey(id), () =>
+      apiClient.getElectionInfo({ electionId: id })
+    );
+  },
+} as const;
+
+async function invalidateElectionQueries(
+  queryClient: QueryClient,
+  electionId: ElectionId
+) {
+  await queryClient.invalidateQueries(getElection.queryKey(electionId));
+  await queryClient.invalidateQueries(getElectionInfo.queryKey(electionId));
```

Comment:
Does `listElections` also need to be invalidated?

Reply:
Going back to the main elections list screen after editing does seem to reflect the changes, but I can invalidate it here just in case.

Reply:
[7576da8](https://github.com/votingworks/vxsuite/pull/5915/commits/7576da8e8fc73c04d023e95cb0d12619f01921b7)